templates/cloudflare-workers/app/root.tsx

```diff
@@ -1,16 +1,11 @@
-import type { LinksFunction } from "@remix-run/cloudflare";
 import {
   Links,
   Meta,
   Outlet,
   Scripts,
   ScrollRestoration,
 } from "@remix-run/react";
-import stylesheet from "./tailwind.css?url";
-
-export const links: LinksFunction = () => [
-  { rel: "stylesheet", href: stylesheet },
-];
+import "./tailwind.css?url";
```

Comment:
For side-effect imports, I don't think you want the `?url` suffix.

```ts
import "./tailwind.css";
```

Reply:
Yep, you're absolutely right, totally missed that when copying all these over

The good thing is testing showed this to pretty obviously be a mistake 😅